nise/generators/ocp/ocp_generator.py

```diff
@@ -323,8 +365,40 @@ def _gen_pods(self, namespaces):
                         "mem_request_gig": mem_request_gig,
                         "mem_limit_gig": mem_limit_gig,
                         "pod_labels": self._gen_openshift_labels(),
+
+
+                    }
+
+                    ros_ocp_data_pods[pod] = {
+                        "namespace": namespace,
+                        "node": node.get("name"),
+                        "resource_id": node.get("resource_id"),
+                        "pod": pod,
+                        "deployment_name":"",
+                        "image_name":"",
```

Comment:
We need to randomly populate `deployment_name` and `image_name` here.

Reply:
done